$OpenBSD: patch-trytond_ir_ir_xml,v 1.1 2019/07/18 07:49:58 semarie Exp $
Backport for issue8375 https://hg.tryton.org/trytond/rev/69c4e6981f14
Evaluate n_sign_posn and p_sign_posn as integer instead of char in XML

Index: trytond/ir/ir.xml
--- trytond/ir/ir.xml.orig
+++ trytond/ir/ir.xml
@@ -19,8 +19,8 @@ this repository contains the full copyright notices an
             <field name="mon_grouping">[3, 3, 0]</field>
             <field name="mon_decimal_point">.</field>
             <field name="mon_thousands_sep">,</field>
-            <field name="p_sign_posn">1</field>
-            <field name="n_sign_posn">1</field>
+            <field name="p_sign_posn" eval="1"/>
+            <field name="n_sign_posn" eval="1"/>
             <field name="positive_sign"></field>
             <field name="negative_sign">-</field>
             <field name="p_cs_precedes" eval="True"/>
